lib/Distribution/Sle/AgamaDevel.pm

```diff
@@ -20,4 +20,9 @@ sub get_grub_menu_installed_system {
             grub_menu_base => Yam::Agama::Pom::GrubMenuBasePage->new()});
 }
 
+sub get_grub_menu_agama {
```

Comment:
you are looking this in the wrong way, now the change is present and should go to lib/Distribution/Opensuse/AgamaDevel.pm, the parent file, the source of true, where the definitive things are located.

Then you need to patch the other classes which didn't receive yet this.
devel for sle and openSUSE , on is mirrow of the other, only difference is the product selected, so those two already have it.

Reply:
updated